openapi/paths/API@bpm@humanTask.yaml

```diff
@@ -7,7 +7,7 @@ get:
 
     - can order on `name`, `priority`, `dueDate`, `state`, `processDefinitionId`, `processInstanceId`, `parentActivityInstanceId`, `assigneeId`, `parentContainerId`, `displayName`, `reachedStateDate`
     - can search on any field that can be used to order results
-    - can filter on `assigned_id`, `state`, `name`, `displayName`, `processDefinitionId`, `caseId`, `rootProcessInstanceId`, `parentProcessInstanceId`
+    - can filter on `assigned_id`, `user_id`, `show_assigned_to_others`, `state`, `name`, `displayName`, `processDefinitionId`, `caseId`, `rootProcessInstanceId`, `parentProcessInstanceId`
```

Comment:
Is it true in all supported version ? (7.11 to today)

Reply:
The filter **show_assigned_to_others** has been just introduced for the 2022.2 (https://github.com/bonitasoft/bonita-web-sp/pull/903)
but the **user_id** was existing (7.11 included)

Reply:
It should be displayed somewhere then (eg: since 2022.2) for `show_assigned_to_others`

Reply:
done